src/Microsoft.OData.Core/ODataPropertyInfo.cs

```diff
@@ -0,0 +1,49 @@
+//---------------------------------------------------------------------
+// <copyright file="ODataPropertyInfo.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData
+{
+    using System.Collections.Generic;
+    using System.Diagnostics.CodeAnalysis;
+    using Microsoft.OData.Edm;
+
+    /// <summary>
+    /// Represents information about a single property of a resource.
+    /// </summary>
+    public class ODataPropertyInfo : ODataItem

```

Comment:
>ODataPropertyInfo [](start = 17, length = 17)

Maybe it's better to make it as abstract class

Reply:
ODataPropertyInfo is used for writing individual primitive values (that are not streams)

---
In reply to: [266659110](https://github.com/OData/odata.net/pull/1204#discussion_r266659110) [](ancestors = 266659110)